packages/reporters/scripts/SendSlackReport.sh

```diff
@@ -25,7 +26,7 @@ pathToFile="./"$fileName
 # Send a curl command to upload the file along with a message to the channel. Store response log for debugging.
 responseFileName="response.json"
 echo "Sending file as slack message $fileName"
-curl https://slack.com/api/files.upload -F token="${SLACK_TOKEN}" -F channels="${SLACK_CHANNEL}" -F title="${messageTitle}" -F fileName="${fileName}" -F file=@"${pathToFile}" | jq '.' >$responseFileName
+# curl https://slack.com/api/files.upload -F token="${SLACK_TOKEN}" -F channels="${SLACK_CHANNEL}" -F title="${messageTitle}" -F fileName="${fileName}" -F file=@"${pathToFile}" | jq '.' >$responseFileName
```

Comment:
Why'd you comment this out?

Reply:
mistake, have corrected. commented it out to test the script from a local docker container